ink-wrapper-types/src/aleph_client.rs

```diff
@@ -1,14 +1,37 @@
-use aleph_client::AsConnection;
-use aleph_client::Balance;
 use aleph_client::{
     pallets::contract::{ContractCallArgs, ContractRpc, ContractsUserApi},
-    SignedConnectionApi, TxInfo, TxStatus,
+    sp_weights::weight_v2::Weight,
+    AsConnection, Balance, CodeHash, ConnectionApi, SignedConnectionApi, TxInfo, TxStatus,
 };
 use anyhow::Error;
 use anyhow::{anyhow, Context, Result};
 use async_trait::async_trait;
 use ink_primitives::AccountId;
 use pallet_contracts_primitives::ContractExecResult;
+use pallet_contracts_primitives::ContractInstantiateResult;
+use scale::Encode;
+use subxt::ext::sp_core::Bytes;
+use subxt::rpc_params;
+
+#[derive(Encode)]
+struct InstantiateRequest {
```

Comment:
Is there no corresponding type for this in the Substrate libraries?

Reply:
I took this one from `cargo-contract`, and it's just defined there in a random spot... Dunno if there's an "official" version, but that suggests to me there isn't

Reply:
Resolved outside of github